src/core/Statiq.App/Commands/CommandUtilities.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Threading;
+using Microsoft.Extensions.Logging;
+
+namespace Statiq.App
+{
+    internal static class CommandUtilities
+    {
+        internal static void WaitForControlC(Action action, ILogger logger)
```

Comment:
Nice work on pulling out the Ctrl-C wait routine - I like the way this abstraction works